examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -98,7 +98,9 @@
 estimators = [
     BayesianRidge(),
     DecisionTreeRegressor(max_features="sqrt", random_state=0),
-    ExtraTreesRegressor(n_estimators=10, random_state=0),
+    ExtraTreesRegressor(
```

Comment:
```suggestion
    # We tuned the hyperparameters of the ExtraTreesRegressor to minimize
    # the execution time
    ExtraTreesRegressor(
```

Reply:
It would be better to emphasize the speed / accuracy tradeoff, e.g.:

> to get a good enough predictive performance for a restricted execution time.